mmdeploy/codebase/mmdet3d/models/mvx_two_stage.py

```diff
@@ -1,41 +1,76 @@
 # Copyright (c) OpenMMLab. All rights reserved.
+from typing import Dict, List, Optional, Sequence
+
+import torch
+
 from mmdeploy.core import FUNCTION_REWRITER
 
 
 @FUNCTION_REWRITER.register_rewriter(
-    'mmdet3d.models.detectors.mvx_two_stage.MVXTwoStageDetector.simple_test')
-def mvxtwostagedetector__simple_test(ctx,
-                                     self,
-                                     voxels,
-                                     num_points,
-                                     coors,
-                                     img_metas,
-                                     img=None,
-                                     rescale=False):
-    """Rewrite this func to remove voxelize op.
+    'mmdet3d.models.detectors.mvx_two_stage.MVXTwoStageDetector.extract_img_feat'  # noqa: E501
+)
+def mvxtwostagedetector__extract_img_feat(ctx, self,
+                                          img: torch.Tensor) -> dict:
+    """Extract features of images."""
+    if self.with_img_backbone and img is not None:
+        if img.dim() == 5 and img.size(0) == 1:
+            img.squeeze_()
+        elif img.dim() == 5 and img.size(0) > 1:
+            B, N, C, H, W = img.size()
+            img = img.view(B * N, C, H, W)
+        img_feats = self.img_backbone(img)
+    else:
+        return None
+    if self.with_img_neck:
+        img_feats = self.img_neck(img_feats)
+    return img_feats
+
+
+@FUNCTION_REWRITER.register_rewriter(
+    'mmdet3d.models.detectors.mvx_two_stage.MVXTwoStageDetector.extract_pts_feat'  # noqa: E501
+)
+def mvxtwostagedetector__extract_pts_feat(
```

Comment:
This rewriter looks like the same as the origin one.

Reply:
removed.